packages/cactus-plugin-consortium-manual/src/main/typescript/plugin-consortium-manual.ts

```diff
@@ -217,9 +218,23 @@ export class PluginConsortiumManual
   public async getConsortiumJws(): Promise<JWSGeneral> {
     const nodes = this.repo.allNodes;
 
+    const { authToken } = this.options;
+
     const requests = nodes
       .map((cnm) => cnm.nodeApiHost)
-      .map((host) => new Configuration({ basePath: host }))
+      .map(function (host) {
+        // if node has token, add to the configuration
+        if (authToken) {
```

Comment:
@elenaizaguirre Similar thoughts here as in my other comment. The if condition is not necessary if we allow the caller to specify the ctor args entirely (with the defaults being provided as before)